notebooks/external_convergence_shear_distributions_tutorial.ipynb

```diff
@@ -0,0 +1,686 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        def sigma8_from_ln10As(ln10As):
</pre><p>this is a definition within a definition that accesses global variables, such as 'camb'. This is dangerous</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSST-strong-lensing/sim-pipeline/pull/64/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>